python/ray/serve/air_integrations.py

```diff
@@ -170,6 +176,43 @@ def split_dict_array(
         return split_list_of_dict
 
 
+@DeveloperAPI
+class SimpleSchemaIngress:
```

Comment:
why is this inlined here?  i.e. why moving this? this can be user facing in its original place right?